### PR TITLE
removed the `xtz` from `Gas` `storage` values

### PR DESCRIPTION
**Pull request by modev0323**

### CODE REVIEW

File: src/utils/general.ts [hunk 1]
```diff
@@ -176,7 +176,7 @@ export function getQueryForDepositsAndRewards(accountid: string) {
 
 export function convertFromUtezToTez(amountInUtez: number) {
   const tezAmount = amountInUtez / 1000000;
-  return `${tezAmount} XTZ`;
+  return tezAmount;
```
Comment (anonymoussprocket) [inline src/utils/general.ts]: This is not correct. The change should be applied only to Gas and Storage fields. For example: block.consumed_gas, operation.gas_limit, operation.storage_limit.

File: src/utils/general.ts [hunk 2]
```diff
@@ -176,7 +176,7 @@ export function getQueryForDepositsAndRewards(accountid: string) {
 
 export function convertFromUtezToTez(amountInUtez: number) {
   const tezAmount = amountInUtez / 1000000;
```
Comment (anonymoussprocket) [inline src/utils/general.ts]: This must use metadata, take a look at the implementation of number formatting in Arronax

Comment (modev0323) [reply src/utils/general.ts]: Used the `Intl.NumberFormat` based on number formatting in `Arronax`

**Comment by vishakh** [comment]
Closing this because we are moving away from TypeScript

**Pull request closed without merging**